src/main/java/pl/edu/zut/mad/schedule/exception/ScheduleExceptionFactory.java

```diff
@@ -0,0 +1,66 @@
+package pl.edu.zut.mad.schedule.exception;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.MessageSource;
+import org.springframework.context.i18n.LocaleContextHolder;
+import org.springframework.stereotype.Component;
+
+@Component
+public class ScheduleExceptionFactory {
+
+    private final MessageSource messageSource;
+
+    @Autowired
+    public ScheduleExceptionFactory(MessageSource messageSource) {
+        this.messageSource = messageSource;
+    }
+
+    public static BadRequestException badRequest(String message) throws BadRequestException {
```

Comment:
I'm not sure that this should be static.

Reply:
This method doesn't require `MessageSource` so it would be more useful to be able to create exception without creating instance of `ScheduleExceptionFactory` if its not needed.